taskt/Core/Automation/Commands/IE Browser Commands/IEElementActionCommand.cs

```diff
@@ -207,6 +174,46 @@ public override void RunCommand(object sender)
                 throw new Exception("Could not find the element!");
             }
         }
+        
+        public override List<Control> Render(frmCommandEditor editor)
+        {
+            base.Render(editor);
+
+            RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_InstanceName", this, editor));
+
+            SearchParameterControls = new List<Control>();
+            SearchParameterControls.Add(CommandControls.CreateDefaultLabelFor("v_WebSearchParameter", this));
+            SearchParameterControls.AddRange(CommandControls.CreateUIHelpersFor("v_WebSearchParameter", this, new Control[] { SearchGridViewHelper }, editor));
+
+            SearchParameterControls.Add(SearchGridViewHelper);
+            RenderedControls.AddRange(SearchParameterControls);
+
+            ElementActionDropdown = (ComboBox)CommandControls.CreateDropdownFor("v_WebAction", this);
+            RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_WebAction", this));
+            RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_WebAction", this, new Control[] { ElementActionDropdown }, editor));
+            ElementActionDropdown.SelectionChangeCommitted += ElementActionDropdown_SelectionChangeCommitted;
+            RenderedControls.Add(ElementActionDropdown);
+
+            ElementParameterControls = new List<Control>();
+            ElementParameterControls.Add(CommandControls.CreateDefaultLabelFor("v_WebActionParameterTable", this));
+            ElementParameterControls.AddRange(CommandControls.CreateUIHelpersFor("v_WebActionParameterTable", this, new Control[] { ElementsGridViewHelper }, editor));
+            ElementParameterControls.Add(ElementsGridViewHelper);
+            RenderedControls.AddRange(ElementParameterControls);
+
+            return RenderedControls;
+        }
+
+        public override string GetDisplayValue()
+        {
+            string parameters = string.Empty;
+            //foreach (DataRow oRow in v_WebActionParameterTable.Rows)
+            foreach (DataRow oRow in v_WebSearchParameter.Rows)
+            {
+                parameters += ", " + oRow["Property Name"] + "=" + oRow["Property Value"];
+            }
+            if (parameters.Length > 0) parameters = parameters.Substring(1);
+            return base.GetDisplayValue() + $" [Perform Action '{v_WebAction} {parameters}' Through IE Instance '{v_InstanceName}']";
```

Comment:
Please change this to: [Perform Action '{v_WebAction} {parameters}' - Instance Name '{v_InstanceName}']

Reply:
done